src/main/java/utilities/CalendarManager.java

```diff
@@ -0,0 +1,158 @@
+package utilities;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.text.DateFormat;
+
+public class CalendarManager {
+    private String dateActualComplete;
+    private Date date;
+    private Calendar calendarDate;
+    private int dates;
+    private DateFormatInput dateFormat;
+
+    public CalendarManager() {
+    }
+
+    /**
+     * Converts string to date according input and format of date.
+     *
+     * @param input is value which will convert to date
+     * @return object type Date
+     */
+    public Date converterStringToDate(final String input) {
```

Comment:
Maybe it could be a better name "convertStringToDate"

Reply:
Thanks for the recommendation